pkgs/development/python-modules/iopath/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, python
+, portalocker
+, termcolor
+, tqdm
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "iopath";
+  version = "v0.1.9";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "facebookresearch";
+    repo = "iopath";
+    rev = version;
+    sha256 = "1kz6xqzxchcjm8fnhd5c0aavh35g379scs45yhrhcclacpzdzrj2";
+  };
+
+  propagatedBuildInputs = [
+    portalocker
+    tqdm
+  ];
+
+  # tests depend on other packages like torch
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "iopath"
+  ];
+
+  meta = with lib; {
+    description = " A python library that provides common I/O interface across different storage backends. ";
```

Comment:
Don’t include a period at the end. (See https://nixos.org/manual/nixpkgs/unstable/#var-meta-description)

You should also remove the leading and trailing white space in description.

Reply:
```suggestion
    description = "Common I/O interface across various storage backends";
```